tests/TestCase/Http/RequestTransformerTest.php

```diff
@@ -259,7 +259,7 @@ public function testToCakeBaseSessionPath()
         $psr = ServerRequestFactory::fromGlobals($server);
         $cake = RequestTransformer::toCake($psr);
 
-        $this->assertEquals('/thisapp', ini_get('session.cookie_path'));
+        $this->assertEquals('/thisapp/', ini_get('session.cookie_path'));
```

Comment:
This was a mistake in the original code. Cookie path names should end in `/` based on the documentation that I could find.